contrib/platform/test/com/sun/jna/platform/win32/PsapiTest.java

```diff
@@ -273,4 +277,42 @@ public void testEnumProcesses() {
         }
         assertTrue("List should contain my pid", foundMyPid);
     }
+
+    @Test
+    public void testQueryWorkingSetEx() {
+        Win32Exception we = null;
+        HANDLE selfHandle = Kernel32.INSTANCE.GetCurrentProcess();
+        MEMORY_BASIC_INFORMATION mbi = new MEMORY_BASIC_INFORMATION();
+        try {
+            SIZE_T bytesRead = Kernel32.INSTANCE.VirtualQueryEx(selfHandle, Pointer.NULL, mbi, new SIZE_T(mbi.size()));
+            assertTrue("It should read correctly", bytesRead.intValue() != 0);
```

Comment:
What's "It"? Let's make the messages stand alone.  And this should be an `assertNotEqual()`.